src/app.js

```diff
@@ -1,21 +1,24 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const PORT = 5000;
+const http = require('http');
 
-module.exports = sum;
+const server = http.createServer((req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+  const pathnames = normalizedURL.pathname
+    .split('/')
+    .filter((path) => path !== '');
+  const query = Object.fromEntries(normalizedURL.searchParams.entries());
+
+  console.log(new URL(req.url, `http://${req.headers.host}`));
```

Comment:
```suggestion
```

Reply:
Set content-type and delete this console log

Reply:
**Fixed**